src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -836,8 +836,11 @@ private synchronized void update()
             adaptiveTrackProjection
                 = new AdaptiveTrackProjection(
                     diagnosticContext,
-                    trackBitrateAllocation.track,
-                    destinationEndpoint::requestKeyframe);
+                    trackBitrateAllocation.track, () ->
+                        destinationEndpoint.getConference().requestKeyframe(
+                            trackBitrateAllocation.endpointID,
+                            trackBitrateAllocation.targetSSRC));
+
```

Comment:
does removing the flexibility of passing the ssrc buy us anything here?  the old call syntax here looked much cleaner, and allowed us to choose which ssrc we requested the keyframe on (which maybe doesn't _happen_ to make a difference since chrome will send on all, but logically seems more correct)

Reply:
if we know the endpoint we don't have to iterate over the endpoints map in conference but i can bring back the old signature if you find it significantly more readable

Reply:
realizing now that i misread this line.  i read it that we were requesting a keyframe from `destinationEndpoint` (which i know realize doesn't make sense), but i see we're just using it to get to the conference.  probably a helper we can write here to help readability, but don't worry about it for this.